pom.xml

```diff
@@ -238,6 +238,8 @@
     <error-prone-support.version>0.14.0</error-prone-support.version>
     <doxia.version>1.12.0</doxia.version>
     <error-prone.configuration-args>
+      <!-- Prefer usage of immutable collectors. -->
```

Comment:
Open question; do we have a section with disabled and enabled checks? Or do we sort them lexicographically (which is what I did right now).

Reply:
Sorting is appreciated.